docs/content/preview/yugabyte-platform/administer-yugabyte-platform/shutdown.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Shut down YugabyteDB Anywhere
+headerTitle: Shut down YugabyteDB Anywhere
+linkTitle: Shut down
+description: Shut YugabyteDB Anywhere down gracefully to perform maintenance.
+menu:
+  preview_yugabyte-platform:
+    identifier: shutdown
+    parent: administer-yugabyte-platform
+    weight: 60
+type: docs
+---
+
+If you need to perform maintenance on the server hosting your YugabyteDB Anywhere installation, requiring a shutdown or restart, you should shut YugabytedB Anywhere down gracefully.
+
+## YBA Installer
+
+If you installed YugabyteDB Anywhere using [YBA Installer](../../install-yugabyte-platform/install-software/installer/#service-management), shut down using the following command:
+
+```sh
+$ sudo yba-ctl stop
+```
+
+The service should automatically start up after a reboot.
```

Comment:
if for whatever reason a vm restart isn't done, `sudo yba-ctl start` will start all the services again